R/mrgsolve.R

```diff
@@ -582,6 +582,8 @@ do_mrgsim <- function(x,
       data$.data_row. <- join_data$.data_row.
       carry.recover <- ".data_row."
       drop <- names(which(!is.numeric(join_data)))
+      # Will be dropped with error later when validating data
+      drop <- drop[!drop %in% c(Pars(x), c(Pars(m), GLOBALS$CARRY_TRAN)]
       data <- data[,setdiff(names(data),drop),drop=FALSE]
```

Comment:
Honestly can't remember why we're dropping data columns here when it will happen later. For now, will keep it as is.